help/en/html/apps/JMRI-FAQ.shtml

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN"
+"http://www.w3.org/TR/html4/strict.dtd">
+
+<html lang="en">
+<head>
+  <meta name="author" content="Jerry Grochow">
+
+  <title>JMRI: Frequently Asked Questions (General)</title>
+
+  <!--#include virtual="/Style.shtml" -->
+
+  <!-- FAQ-Head -->
+    <script type="text/javascript">
+    /*<![CDATA[*/document.documentElement.className="hasJS";/*]]>*/
+  </script>
+    <link rel="stylesheet" type="text/css" href="/web/css/faq.css"
+          media="screen">
+  <!-- /FAQ-Head -->
+
+</head>
+
+<body>
+  <!--#include virtual="/Header.shtml" -->
+  <div id="mBody">
+    <!--#include virtual="Sidebar.shtml" -->
+    <div id="mainContent">
+
+      <h1>JMRI: Frequently Asked Questions (General)</h1>
+            <h2><em>Work in Progress</em></h2>
+
+
+    <div id="para">
+
+      <p>This page is the JMRI general "frequently asked questions" list. Items listed here if they are asked in user forums,
+      of experienced users, and of developers, even if the same information is available elsewhere in the JMRI window help and 
+      on the JMRI web site.</p>
+      
+      <p>Additional FAQs are also avilable throughout the JMRI Help/Web site:</p>
+      <ul>
+        <li><a href="PanelPro/FAQ.shtml">PanelPro FAQ</a>
+        <li><a href="PanelPro/GettingStarted.shtml">Getting Started With PanelPro</a>
+        <li><a href="DecoderPro/FAQ.shtml">DecoderPro FAQ</a>
+        <li><a href="DecoderPro/Tour.shtml">DecoderPro Quick Tour</a>
+        <li><a href="StoringYourWork.shtml">Storing Your Work</a>
+        <li><a href="../tools/scripting/HowTo.shtml">Scripting "How To"</a>
+      </ul>      
+      
+
+      <dl class="faq">
+        <dt class="on" id="canuse">Can I use JMRI to...</dt>
+
+        <dd>
+        <ul>
```

Comment:
I think this list should be dictionary list instead of a bulleted list.

Reply:
I’m not a super expert in HTML so I basically follow models I find in other JMRI pages.  I looked up a “dictionary list” online but couldn’t find it.  I looked up dt/dl but they seem to be something different: “The “dt“ tag defines a term/name in a description list.  The “dt” tag is used in conjunction with “dl” (defines a description list) and “dd” (describes each term/name).”  Thanks.

Reply:
Sorry, you're right, I meant "description list"

Reply:
Both suggestions implemented.